packages/flutter_tools/lib/src/dart/dependencies.dart

```diff
@@ -7,6 +7,22 @@ import 'package:analyzer/analyzer.dart' as analyzer;
 import '../base/file_system.dart';
 import '../dart/package_map.dart';
 
+// List of flutter specific environment configurations.
+// See https://github.com/munificent/dep-interface-libraries/blob/master/Proposal.md
+final List<String> _configurationConstants = <String>['dart.library.io'];
```

Comment:
We could potentially populate this list with all dart: libraries that flutter supports.